package.json

```diff
@@ -8,7 +8,7 @@
     "dev-tauri-renderer": "vite -c vite.config.tauri.ts",
     "build-tauri": "npm run build-tauri-renderer && tauri build",
     "dev-tauri": "tauri dev",
-    "dev-chromium": "NODE_ENV=development vite build -c vite.config.chromium.ts --watch",
+    "dev-chromium": "vite -c vite.config.chromium.ts",
     "dev-firefox": "NODE_ENV=development vite build -c vite.config.firefox.ts --watch",
```

Comment:
this command can be changed to the real dev command?

Reply:
what's the "real dev command"?

Reply:
like dev-chromium

Reply:
firefox not support that 😂
https://github.com/samrum/vite-plugin-web-extension/issues/87#issuecomment-1501236239